### DISCUSSION TITLE
Power Users & "Overwhelmed by notifications" prompt...

### DISCUSSION THREAD

**Discussion by BenjamenMeyer** [community/community#12976] [Other Feature Feedback, Questions, & Ideas] [answered]
Saw [#58](https://github.com/github/feedback/discussions/58); but couldn't respond there.

The solution provided in the aforementioned discussion was the "GroupBy:repository" feature. I use "Group By: Repository" all the time - love it, and it was a great feature of the old UI so I really appreciated it when they added that functionality back in the new UI.

I understand why GH wants to prompt users about being "overwhelmed by notifications", so I'm not suggesting it entirely go away. However, for those of us that are GH power users where we follow, monitor, and track many repositories (work related, open source activities, side projects, etc) we know and expect to get a ton of notifications.

Presently, I get that prompt every few days - I live on the notification UI as a means to watch for PRs to respond to and issues to review. I mostly ignore it, but I've acknowledged it a few times to try to get it to go away; but it keeps coming back.

It'd be great if there was a way to tell GH to never show that prompt again.

**Comment by omerbensaadon** [comment, marked as answer]
We unshipped this banner :)
